src/styles/utils/bem-block.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__img {
+    width: 100%;
+    margin-bottom: 40px;
+  }
+  &__prod {
+    margin-bottom: 16px;
+  }
+  &__rewie {
+    margin-bottom: 24px;
+  }
+  &__price {
+    margin-bottom: 16px;
+  }
+}
+
+.product {
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%;
+    color: v.$col-blue;
+    margin-bottom: 4px;
+  }
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    color: v.$col-grey;
+  }
+}
+
+.rewiews {
```

Comment:
The class name 'rewiews' contains a typographical error. It should be 'reviews'.